src/app/views/company/settings/organization-settings.directive.coffee

```diff
@@ -20,17 +19,18 @@ DashboardOrganizationSettingsCtrl = ($scope, $window, MnoeOrganizations, Utiliti
     angular.copy(organization, $scope.origModel)
     $scope.isLoading = false
 
-  # Save the current state of the credit card
+  # Save the current state of the organization settings
   $scope.save = ->
     $scope.isLoading = true
+    # Update organization
     MnoeOrganizations.update($scope.model).then(
       (response) ->
         $scope.errors = ''
-        angular.copy(response.organization, $scope.model)
-        angular.copy(response.organization, $scope.origModel)
+        angular.merge(response.organization, $scope.model)
```

Comment:
What was the issue with `copy`?
I'd assume it'd be safer to refresh the record with the backend response?